hummingbot/connector/exchange/coinzoom/coinzoom_exchange.py

```diff
@@ -103,6 +104,10 @@ def __init__(self,
         self._trading_rules_polling_task = None
         self._last_poll_timestamp = 0
         self._throttler = Throttler(rate_limit = (8.0, 6))
+        self._real_time_balance_update = False
+        self._update_balances_fetching = False
+        self._update_balances_queued = False
```

Comment:
```suggestion
        self._update_balances_fetching = False
        self._update_balances_queued = False
```
I think we should remove this to simplify balance updates.